be/src/storage/lake/persistent_index_sstable.h

```diff
@@ -55,6 +55,8 @@ class PersistentIndexSstable {
 
     sstable::Iterator* new_iterator(const sstable::ReadOptions& options) { return _sst->NewIterator(options); }
 
+    const PersistentIndexSstablePB sstable_pb() const { return _sstable_pb; }
```

Comment:
```suggestion
    const &PersistentIndexSstablePB sstable_pb() const { return _sstable_pb; }
```

if is not expecting caller to change the PB, const reference return value would be preferred.